### ISSUE TITLE
Could not find a version that satisfies the requirement tgtg

### ISSUE THREAD

**Issue by christoschronopoulos** [ahivert/tgtg-python#100]
Dear all, I have installed pypi and I am using "pip install tgtg" but I get the following error
"Could not find a version that satisfies the requirement tgtg (from versions: )
No matching distribution found for tgtg"
What do I do wrong?

**Comment by ahivert** [comment]
On which python version are you trying to install tgtg ? You have to use at least python 3.6

**Comment by christoschronopoulos** [comment]
Thank you for responding to this issue. I managed to solve it by running the following command:
python3 -m pip install tgtg

which gave this result at the last line
"Successfully installed requests-2.25.1 tgtg-0.5.0"

After that, I ran the following commands:
sudo pipenv --three
sudo pipenv install 
sudo pipenv shell
from tgtg import TgtgClient

However, trying to import from tgtg gave this error:
ModuleNotFoundError: No module named 'tgtg'

This is where I am stuck now. Not sure how to proceed. Any help would be greatly appreciated.
